adapters/couchdb.md

```diff
@@ -0,0 +1,82 @@
+# CouchDB CloudEvents Adapter
+
+This document describes how to convert:
+- [CouchDB Document events](http://docs.couchdb.org/en/stable/api/database/changes.html) and
+- [CouchDB Database events](http://docs.couchdb.org/en/stable/api/server/common.html#db-updates)
+
+into CloudEvents.
+
+Each section below describes how to determine the CloudEvents attributes
+based on the event type.
+
+## /db/_changes
+
+### Update
+
+| CloudEvents Attribute | Value                                    |
+| :-------------------- | :--------------------------------------- |
+| `id`                  | The event sequence identifier (`seq`) |
+| `source`              | The server URL / db_name             |
+| `specversion`         | `1.0`                                |
+| `type`                | `org.apache.couchdb.document.update`     |
+| `datacontenttype`     | `application/json`                       |
+| `dataschema`          | Omit                                     |
+| `subject`             | `id` value                           |
```

Comment:
I assume this is the "Id" of the document that changed

Reply:
yes. Updating...